src/plugins/intel_gpu/src/graph/impls/ocl/roll.cpp

```diff
@@ -1,14 +1,8 @@
 // Copyright (C) 2022 Intel Corporation
 // SPDX-License-Identifier: Apache-2.0
 //
-
-// We have problem with includes when ENABLE_ONEDNN_FOR_GPU is OFF,
```

Comment:
It would be nice to remove such comments from openvino/src/plugins/intel_gpu/src/graph/impls/ocl/bucketize.cpp too

Reply:
done